### DISCUSSION TITLE
"Pore-wall flux [mol.m-2.s-1]":  data from DFN model simulation

### DISCUSSION THREAD

**Discussion by manjunathnilugal** [pybamm-team/PyBaMM#2266] [Q&A]
Dear All, 
I have performed the simulation for the Lithium ion battery and I want to save the data for the
 "**Pore-wall flux [mol.m-2.s-1] for both the anode and cathode near separator and current collector**" from DFN model simulation.
I tried search in the ***model.variables.search("flux")***, but didn't get the solution.
Kindly help me, if its possible to get these variables from the simulation.

![image](https://user-images.githubusercontent.com/82285213/186946882-65bb88d8-ba09-4e21-8c96-05ebc54fd4bb.png)

**Comment by HassanSewailem** [comment]
I have the same question too.
